libraries/standard/http/src/http_client.c

```diff
@@ -316,29 +336,45 @@ static HTTPStatus_t addHeader( HTTPRequestHeaders_t * pRequestHeaders,
                HTTP_HEADER_LINE_SEPARATOR_LEN +
                HTTP_HEADER_LINE_SEPARATOR_LEN;
 
-    /* If we have enough room for the new header line, then write it to the header buffer. */
+    /* If we have enough room for the new header line, then write it to the
+     * header buffer. */
     if( ( backtrackHeaderLen + toAddLen ) <= pRequestHeaders->bufferLen )
     {
         /* Write "<Field>: <Value> \r\n" to the headers buffer. */
 
         /* Copy the header name into the buffer. */
-        memcpy( pBufferCur, pField, fieldLen );
+
+        /* MISRA rule 21.15 flags the memcpy of two different types, const uint8_t*
+         * and const char*. These types are the same size, so this comparision is
```

Comment:
`pBufferCur` is of type `uint8_t *` and `pField` is of type `const uint8_t *` - Is this comment correct or did I miss anything?

Reply:
Excuse me I went copied this in too many places.